data/zilliqa/assets/zil1nfuy0fuvdun80m6f5vxfqycz0caphpf8tksmlt/meta.json

```diff
@@ -2,7 +2,9 @@
 	"name": "Dope Crew Friends",
 	"symbol": "DCF",
 	"links": {
-		"github": "dopecrewfriends",
+		"web": "www.dopecrewfriends.com",
+		"instagram": "www.instagram.com/daehandope/"
+		"telegram": "https://t.me/DopeCrewFriends"
```

Comment:
this is not a valid json, go to an online json validator

Reply:
i just checked again, jsonlint shows 'valid json file'
cant seem to figure out what could be wrong with it now. 
let me know if you see something specifically wrong with meta.json

Reply:
moved 'web' out of links
currently still valid on jsonlist.com. 

Please help locating the issue.

Reply:
use another one, I don't have time for that